### ISSUE TITLE
Official Chia Blockchain discord server + Chia coin Miners Group.

### ISSUE THREAD

**Issue by Miahelloworld** [Miahelloworld/chia-blockchain-discord-server#1]
discord link: https://discord.gg/Mz4SG4KrYN

![chia network discord server](https://user-images.githubusercontent.com/72310174/114150274-72f3a100-994e-11eb-8fea-71eb80e0ad8a.png)

[Suggestion] Official Chia Blockchain discord server + Chia coin Miners.
Hey guys,

Just a suggestion for the community, I think it would be a great benefit to have a discord for the Chia blockchain. There is one here discord link: https://discord.gg/Mz4SG4KrYN  but it seems to be lacking membership.

I know there is an official keybase but discord offers so much more and is completely free. much more moderation options and general flexibility. A lot of other crypto communities have migrated over to it as well so that is another reason to move IMO.

If the slack is going to remain I'd also suggest including it in the subreddit sidebar.

None of the social channels linked in the sidebar offer IM functionality. Now that we have a quite large user base IM, real-time chats are of great benefit and will prevent social channels like Reddit from getting clogged up with the same questions over and over.

![Official Chia Blockchain discord server + Chia coin Miners](https://user-images.githubusercontent.com/72310174/114149968-12646400-994e-11eb-8eaf-80d6c8948f8a.png)

**Comment by SlinkyPotato** [comment]
Agree, it is super convenient to have an official discord channel. That way there is a single location for the many crypto projects discussions that's going on.

**Comment by codephunk** [comment]
Hi there. I got banned and blocked from the Chia Discord server without further notice. Is there a support email or a administrative Discord user i can contact regarding this issue?

**Comment by Miahelloworld** [comment]
> Hi there. I got banned and blocked from the Chia Discord server without further notice. Is there a support email or a administrative Discord user i can contact regarding this issue?

May be you can apply a new discord account, and join again.

**Comment by rcamarda390** [comment]
> > Hi there. I got banned and blocked from the Chia Discord server without further notice. Is there a support email or a administrative Discord user i can contact regarding this issue?
> 
> May be you can apply a new discord account, and join again.

as far as I can tell, there are not admins nor modirators online.
I'd like to see added channels, RasperryPi and CLI

**Comment by rcamarda390** [comment]
> > > Hi there. I got banned and blocked from the Chia Discord server without further notice. Is there a support email or a administrative Discord user i can contact regarding this issue?
> > 
> > 
> > May be you can apply a new discord account, and join again.
> 
> as far as I can tell, there are not admins nor modirators online.
> I'd like to see added channels, RasperryPi and CLI

I checked this morning and I see 2 new channels
Thanks!

**Comment by alphak3y** [comment]
> Hi there. I got banned and blocked from the Chia Discord server without further notice. Is there a support email or a administrative Discord user i can contact regarding this issue?

I had the same issue. No DM. No warning. Nothing.

**Comment by Miahelloworld** [comment]
> > Hi there. I got banned and blocked from the Chia Discord server without further notice. Is there a support email or a administrative Discord user i can contact regarding this issue?
> 
> I had the same issue. No DM. No warning. Nothing.

You may be got banned because of spam. You can apply a new discord account, and join again.

**Comment by alphak3y** [comment]
I would like to make a suggestion that you do Server Mute for 72 hours to let the person know there is an issue.

This will create a better unity for the Chia community and avoid any confusion :)

**Comment by Nomad965** [comment]
I got banned for no reason as well, no DM no warning nothing...

**Comment by Miahelloworld** [comment]
> I got banned for no reason as well, no DM no warning nothing...

If you send too many ads and spam, you will be got banned. You can apply a new discord account, and join again.

**Comment by Nomad965** [comment]
> > I got banned for no reason as well, no DM no warning nothing...
> 
> If you send too many ads and spam, you will be got banned. You can apply a new discord account, and join again.

I never sent anything, actually I was helping other users with support, the only thing I did was reference swars plots manager discord server for one user that needed help.
